### PR TITLE
r/aws_fms_policy - add description

### LINKED ISSUE #29873: [Enhancement]: Firewall Manager does not have Description
### Description

The `Firewall Manager Resource` does not include an option to create a description for the Firewall Manager Policy even though the API permits.

### Affected Resource(s) and/or Data Source(s)

aws_fms_policy

### Potential Terraform Configuration

```terraform
resource "aws_fms_policy" "edge_services" {
  name                               = "Edge-Services"
  description                        = "Edge Services"
}
```

### References

* https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/fms_policy
* https://docs.aws.amazon.com/fms/2018-01-01/APIReference/API_PutPolicy.html

### Would you like to implement a fix?

No

### PR DESCRIPTION
**Pull request by bennylu2**
### Description
Add `description` to policy

### Relations
Closes #29873

### References
https://docs.aws.amazon.com/fms/2018-01-01/APIReference/API_Policy.html

Note: I was unable to run tests, it requires management account 

```
=== RUN   TestAccFMS_serial/Policy/basic
    acctest.go:914: this AWS account must be the management account of an AWS Organization
```

### CODE REVIEW

File: website/docs/r/fms_policy.html.markdown [hunk 1]
```diff
@@ -55,6 +55,7 @@ The following arguments are supported:
 * `name` - (Required, Forces new resource) The friendly name of the AWS Firewall Manager Policy.
 * `delete_all_policy_resources` - (Optional) If true, the request will also perform a clean-up process. Defaults to `true`. More information can be found here [AWS Firewall Manager delete policy](https://docs.aws.amazon.com/fms/2018-01-01/APIReference/API_DeletePolicy.html)
 * `delete_unused_fm_managed_resources` - (Optional) If true, Firewall Manager will automatically remove protections from resources that leave the policy scope. Defaults to `false`. More information can be found here [AWS Firewall Manager policy contents](https://docs.aws.amazon.com/fms/2018-01-01/APIReference/API_Policy.html)
+* `description` - (Optional) The definition of the AWS Network Firewall firewall policy.
```
Comment (kderck) [inline website/docs/r/fms_policy.html.markdown:58]: Do you mean description over definition?

Comment (bennylu2) [inline website/docs/r/fms_policy.html.markdown:58]:
```suggestion
* `description` - (Optional) The description of the AWS Network Firewall firewall policy.
```

Comment (bennylu2) [reply website/docs/r/fms_policy.html.markdown]: The aws docs say [definition](https://docs.aws.amazon.com/fms/2018-01-01/APIReference/API_Policy.html#:~:text=Required%3A%20No-,PolicyDescription,-The%20definition%20of) but description makes more sense. Updated, thanks for feedback

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

```console
% AWS_DEFAULT_REGION=us-east-1 make testacc TESTARGS='-run=TestAccFMS_serial/Policy/basic' PKG=fms 
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/fms/... -v -count 1 -parallel 20  -run=TestAccFMS_serial/Policy/basic -timeout 180m
=== RUN   TestAccFMS_serial
=== PAUSE TestAccFMS_serial
=== CONT  TestAccFMS_serial
=== RUN   TestAccFMS_serial/Policy
=== RUN   TestAccFMS_serial/Policy/basic
--- PASS: TestAccFMS_serial (1231.16s)
    --- PASS: TestAccFMS_serial/Policy (1231.16s)
        --- PASS: TestAccFMS_serial/Policy/basic (1231.16s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/fms	1236.498s
```

**Comment by ewbankkit** [comment]
@bennylu2 Thanks for the contribution :tada: :clap:.

**Pull request merged**